UPGRADING.md

```diff
@@ -1,5 +1,13 @@
 # Upgrade guidelines
 
+## 0.38.0 to 0.39.0
+
+(#325 Logs collection is now disabled by default for Splunk Observability
+destination](https://github.com/signalfx/splunk-otel-collector-chart/pull/325)
+
+If you send logs to Splunk Observability destination, make sure to set
+`splunkObservability.logsEnabled` to true.
```

Comment:
could be helpful to show an install w/ `--set` example and/or clarify that this belongs in a values file w/ example.

Reply:
Makes sense. Updated